servicetalk-grpc-netty/src/test/java/io/servicetalk/grpc/netty/ProtocolCompatibilityTest.java

```diff
@@ -610,6 +611,50 @@ void clientH2ReturnStatus() throws Exception {
         }
     }
 
+    @Test
+    void grpcJavaToServiceTalkUnimplementedService() throws Exception {
+        try (TestServerContext server = serviceTalkServerBlocking(ErrorMode.STATUS, false, null, null);
+             CompatClient client = grpcJavaClient(server.listenAddress(), null, false, null)) {
+            final Single<CompatResponse> response =
+                    client.unimplementedServerCall(CompatRequest.newBuilder().setId(1).build());
```

Comment:
Instead of adding a new `unimplementedServerCall` in the proto, consider adding service parameter for `serviceTalkServerBlocking`, `serviceTalkServer`, and `grpcJavaServer`. There you can pass an unimplemented service like `new BlockingCompatService() { }`. That way you can validate all 4 API flavors in every test method.

Reply:
Always throwing will still break the default behavior, especially in cases where the user has implemented the `GrpcResponseWriter` variant.

Reply:
Looks like a reply to a wrong thread, but you are right. We should still skip those that already have a default.